README.md

```diff
@@ -1,12 +1,13 @@
-# Thrustmaster T150 Force Feedback Wheel Linux drivers
+# Thrustmaster T150 and TMX Force Feedback Wheel Linux drivers
```

Comment:
I'm unsure now if then the driver should still be called t150, but let's keep it for now

Reply:
It's fairly normal to maintain terminology even as the scope increases in kernel drivers.